cypress/components/help/help.cy.tsx

```diff
@@ -54,12 +39,12 @@ context("Tests for Help component", () => {
       getComponent("help").should("have.id", id);
     });
 
-    it.each(["-1", "0", "1", "5"])(
+    it.each([-1, 0, 1, 5])(
       "should check %s as tabIndex for Help component",
       (tabIndex) => {
         CypressMountWithProviders(<HelpComponent tabIndex={tabIndex} />);
 
-        if (tabIndex === "-1") {
+        if (String(tabIndex) === "-1") {
```

Comment:
I think we can go with
```suggestion
        if (tabIndex == "-1") {
```

Reply:
Hi @DlgSHi , I've tried your suggested solution but didn't work me. Please let me know if we can fix this using some different method.

Reply:
changed as suggested locally and it works as expected